bin/run_unittests.sh

```diff
@@ -57,22 +77,70 @@ fi
 
 randomize_redis_port
 
-set -e
```

Comment:
I just removed this command because it is already defined at the beginning of the file, in line 3.